# HG changeset patch
# User Ben Abbot  <bpabbott@mac.com>
# Date 1450844918 18000
#      Tue Dec 22 23:28:38 2015 -0500
# Node ID 93a48264f402c4a56eddf1462b0a8a77f4fa1db1
# Parent  6ac3d299c5ad5bba7e948003fc35a5b03ef22c1c
Remove erroneous TODO introduced by rev 6ac3d299c5ad.

* __gnuplot_print__.m: Remove TODO.

diff --git a/scripts/plot/util/private/__gnuplot_print__.m b/scripts/plot/util/private/__gnuplot_print__.m
--- a/scripts/plot/util/private/__gnuplot_print__.m
+++ b/scripts/plot/util/private/__gnuplot_print__.m
@@ -111,18 +111,16 @@ function opts = __gnuplot_print__ (opts)
       local_drawnow (["svg dynamic " gp_opts], opts.name, opts);
     case {"aifm", "corel", "eepic", "emf", "fig"}
       local_drawnow ([opts.devopt " " gp_opts], opts.name, opts);
     case {"cairolatex", "epscairo", "epscairolatex", ...
           "epscairolatexstandalone", "pdfcairo", "pdfcairolatex", ...
           "pdfcairolatexstandalone", "pngcairo"}
       term = opts.devopt;
       if (strfind (term, "standalone"))
-        ## TODO: Specifying the size of the figure and page are not yet
-        ## supported. Specifying the font size also does not work.
         gp_opts = sprintf ("standalone %s", gp_opts);
         term = strrep (term, "standalone", "");
       endif
       if (strfind (term, "epscairolatex"))
         gp_opts = sprintf ("eps %s", gp_opts);
         term = strrep (term, "epscairolatex", "cairolatex");
       elseif (strfind (term, "pdfcairolatex"))
         gp_opts = sprintf ("pdf %s", gp_opts);
